editor/src/components/inspector/sections/component-section/data-selector-cartouche.tsx

```diff
@@ -13,12 +13,13 @@ import type { DataPickerOption } from './data-picker-utils'
 interface DataPickerCartoucheProps {
   data: DataPickerOption
   selected: boolean
+  forceRole?: CartoucheUIProps['role']
```

Comment:
I'd be tempted to call this `forcedRole`, because the current name makes me think it's a Boolean.